### ISSUE TITLE
Autostart by Default

### ISSUE THREAD

**Issue by xenstar** [erickutcher/httpdownloader#301]
Can you create an option so that from settings, we can select, and it can automatically start when windows start?

**Comment by erickutcher** [comment]
There's a setting in the Advanced Options that'll let you "Resume previously downloading files upon startup".

If you're referring to having the program start when Windows starts, then the best solution is to add a shortcut of the program to your Startup folder. It should be located here: `C:\Users\[USERNAME]\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup`

**Comment by xenstar** [comment]
Yes, but sometimes I ask my noob friends to use this as download manager, if they can directly do from settings, or it's turn on by default, it would be great.

**Comment by erickutcher** [comment]
A setting for that would require creating a registry entry which I don't really want to use. Maybe someday. The Startup folder is still pretty easy though. Just have them navigate to that folder and then copy a shortcut to it. Makes things easy to clean up if you decide to remove the program since you won't have to go into the registry. Everything would just be in standard folders on the system.
